code/game/gamemodes/dynamic/dynamic_midround_rolling.dm

```diff
@@ -39,7 +39,7 @@
 	var/list/drafted_heavies = list()
 	var/list/drafted_lights = list()
 
-	for (var/datum/dynamic_ruleset/midround/ruleset in midround_rules)
+	for (var/datum/dynamic_ruleset/midround/ruleset in shuffle(midround_rules))
```

Comment:
This also only adds them to a list that gets picked randomly from using draft midround ruleset.
If these weren't random, then dynamic would be entirely deterministic and we would have the exact same gamemode every single round